test/util/cfssl/ca-config.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Let's add a link to the relevant cfssl docs at the top here.

Reply:
JSON doesn't support comments